src/features/student/selfKnowledge/queries/self-knowledge.query/self-knowledge.query.ts

```diff
@@ -185,3 +186,21 @@ export function useRemoveSelfKnowledgeCategoryMutation ({ onError, onSuccess }:
     onError
   })
 }
+
+export interface DeleteSelfKnowledgeElementsVariables {
+  selfKnowledgeElementIds: string[]
+}
+
+export function useDeleteSelfKnowledgeElementsMutation ({ onError, onSuccess }: MutationArgs<string, DeleteSelfKnowledgeElementsVariables> = {}) {
+  const invalidateSelfKnowledgeCommonQuery = useInvalidateQuery([...selfKnowledgeCommonQueryKey])
```

Comment:
pas besoin d'invalidate toutes les queries 

```suggestion
  const invalidateSelfKnowledgeCommonQuery = useInvalidateQuery([...selfKnowledgeElementsQueryKey])
```